packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useCodeComparisonStyles = makeStyles({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    '> *': {
+      width: '100%',
+    },
+
+    '> :not(:last-child)': {
+      marginRight: '5%',
+    },
+  },
+});
+
+export const CodeComparison = (props: { children: React.ReactElement[] }) => {
+  const { children } = props;
+  const classes = useCodeComparisonStyles();
+
+  return <div className={classes.root}>{children}</div>;
+};
+
+const codeLanguages: Record<string, string> = {
```

Comment:
what about this explicit type annotation? I suggested to remove it.

Reply:
Here's what I meant on our previous conversation.

![image](https://user-images.githubusercontent.com/39736248/119648066-4e318b00-be21-11eb-9eca-4bb08769a39b.png)